girder/utility/server.py

```diff
@@ -20,10 +20,12 @@
 import cherrypy
 import mimetypes
 import os
+import six
 
 import girder.events
 from girder import constants
 from girder.utility import plugin_utilities, model_importer
+from girder.utility.plugin_utilities import _plugin_webroots  # noqa
```

Comment:
If we need access to this from outside the module, let's make an accessor method `getPluginWebroots()` in the public namespace.